packages/markdown/styles/main.scss

```diff
@@ -0,0 +1,5 @@
+@import '~@readme/syntax-highlighter/node_modules/codemirror/lib/codemirror.css';
```

Comment:
I don't know if this accessor will work when installed through npm.

Reply:
I believe the `~` prefix is a standard `sass-loader` feature which points to the `node_modules/` directory, so conceptually it _should_ work... (I've had no problems with it thus far!)

Reply:
Sorry, this was in reference to the accessor containing `/node_modules/codemirror/lib/codemirror.css` after `@readme/syntax-highlighter`. That said, I've been thinking about this since last week and I think this is okay, actually. 

I was thinking that since when you would install `@readme/markdown`, depending on your version of npm you would either get:

* `@readme/syntax-highlighter` and `codemirror` installed alongside each other in `node_modules/`.
* `@readme/syntax-highlighter` installed in `node_modules/` and then another `node_modules/` directory inside `@readme/syntax-highlighter/node_modules` that contained `codemirror`.

However none of that matters because as since we're using Lerna here, Lerna takes care of the package symlinks so  `@readme/syntax-highlighter/node_modules/codemirror/lib/codemirror.css` exists, but also since we build this package, we aren't loading `styles/main.scss`.

All good!